api/src/main/java/org/pmiops/workbench/api/WorkspacesController.java

```diff
@@ -979,4 +978,15 @@ private void recordOperationTime(Runnable operation, String operationName) {
         new WorkspaceCreatorFreeCreditsRemainingResponse()
             .freeCreditsRemaining(freeCreditsRemaining));
   }
+
+  private DbWorkspace loadDbWorkspace(String workspaceNamespace, String workspaceId) {
```

Comment:
I think `workspaceService.getRequired` is equivalent - can just switch the other calls to using this instead: https://github.com/all-of-us/workbench/blob/93ebc23a007925db89fa71d3451143a611227d6f/api/src/main/java/org/pmiops/workbench/workspaces/WorkspaceServiceImpl.java#L272-L278

Reply:
that;s good know. Didn't know that is a method called that before.